distribution/test/itests/test-itests-ddf/pom.xml

```diff
@@ -178,6 +178,11 @@
             <classifier>features</classifier>
             <type>xml</type>
         </dependency>
+        <dependency>
+            <groupId>io.fastjson</groupId>
```

Comment:
Is this dependency needed here?  It looks like it is not injected into the Pax Exam container.